### PR TITLE
[DI2-374] Move all construction file code into a single place

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: src/app/pages/ConstructionFilePage/components/ImageViewer/ImageViewer.test.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { render } from '@testing-library/react'
-import withAppContext from '../../utils/withAppContext'
+import withAppContext from '../../../../utils/withAppContext'
 import ImageViewer, { IMAGE_VIEWER_TEST_ID } from './ImageViewer'
 
 jest.mock('openseadragon')
 
+// TODO: This file should really have a lot more tests.
```
Comment (jonkoops) [inline src/app/pages/ConstructionFilePage/components/ImageViewer/ImageViewer.test.tsx:7]: These tests still need to be expanded, but the PR can be reviewed.

File: src/reducers/root.ts [hunk 2]
```diff
@@ -40,7 +39,6 @@ const rootReducer = (routeReducer: any) => (oldState: any = {}, action: any) =>
     [DATA_SEARCH_REDUCER]: DataSearchReducer,
     [SELECTION]: SelectionReducer,
     [DATA_SELECTION]: DataSelectionReducer,
-    [FILES]: filesReducer,
```
Comment (chrisvanmook) [inline src/reducers/root.ts:43]: 🎉

**Review by chrisvanmook — APPROVED** [review]

File: src/reducers/root.ts (hunk 2 above)
Comment (jonkoops) [reply src/reducers/root.ts:43]: Yeah, we are getting there :100:

File: src/app/pages/ConstructionFilePage/ConstructionFilePage.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+const ConstructionFilePage: FunctionComponent = () => {
+  const history = useHistory()
+  const { setDocumentTitle } = useDocumentTitle()
+  const { id } = useParams<ConstructionFilePageParams>()
+  const [fileName] = useParam(fileNameParam)
+  const [fileUrl] = useParam(fileUrlParam)
+  const result = usePromise(() => getBouwdossierById(id), [id])
+
+  useEffect(() => {
+    if (fileName) {
```
[the remark this answers is not in the document]
Comment (jonkoops) [reply src/app/pages/ConstructionFilePage/ConstructionFilePage.tsx]: Fixed

**Pull request merged**